packages/terra-core-docs/CHANGELOG.md

```diff
@@ -6,6 +6,16 @@
   * Added Higher Order examples for `terra-profile-image`.
   * Added Accessibility Guide for `terra-Profile-image`.
 
+* Changed
+  * Added Accessibility Guide for `terra-Profile-image`.
+
+* Changed
+  * Updated `terra-list` section guide to not use listbox role.
+
+* Changed
+  * Added Higher Order examples for `terra-profile-image`.
+  * Added Accessibility Guide for `terra-Profile-image`.
+
```

Comment:
All of these can be under a single `Changed` heading (Not sure how it got overlooked in previous releases).